src/Microsoft.Fast.Components.FluentUI/Components/NavMenu/FluentNavMenuLink.razor.cs

```diff
@@ -91,22 +97,21 @@ public partial class FluentNavMenuLink : FluentComponentBase
 
     internal bool HasIcon => !string.IsNullOrWhiteSpace(Icon) || IconContent is not null;
 
-    [CascadingParameter(Name = "NavMenuExpanded")]
-    private bool NavMenuExpanded { get; set; }
-
     public FluentNavMenuLink()
     {
         Id = Identifier.NewId();
     }
 
-    protected override void OnParametersSet()
+    protected override void OnInitialized()
     {
+        base.OnInitialized();
```

Comment:
We want to add these only once when the component is created, not every time its parameters are set.